force/sobjects.go

```diff
@@ -112,6 +114,41 @@ func (forceApi *ForceApi) DeleteSObject(id string, in SObject) (err error) {
 	return
 }
 
+func idsFromURIs(uris []string) []string {
+	ids := make([]string, len(uris))
+	for i, uri := range uris {
+		parts := strings.Split(uri, "/")
+		ids[i] = parts[len(parts)-1]
+	}
+	return ids
+}
+
+func (forceApi *ForceApi) GetSFIDsByExternalId(apiName, externalKey, externalId string) ([]string, int, error) {
+	uri := fmt.Sprintf("%v/%v/%v", forceApi.apiSObjects[apiName].URLs[sObjectKey], externalKey, externalId)
+
+	params := url.Values{"fields": []string{"Id"}}
+	sobj := sobjects.BaseSObject{}
+
+	statusCode, err := forceApi.Get(uri, params, &sobj)
+	if err == nil {
+		return []string{sobj.Id}, statusCode, nil
+	}
+
+	if statusCode != 300 {
```

Comment:
Can we make 300 a const variable so that there's better understanding of why this is special?

Reply:
(through naming)